clients/benchmarks/CMakeLists.txt

```diff
@@ -231,9 +231,22 @@ if (NOT WIN32)
 endif()
 
 if (NOT WIN32)
-  if (BUILD_WITH_ROCTX)
-    target_link_libraries(rocsparse-bench PRIVATE -lroctx64)
+  find_package(roctx)
+
+  if(ROCTX_FOUND)
+    target_link_libraries(rocsparse-bench PRIVATE roctx::roctx)
     target_compile_definitions( rocsparse-bench PRIVATE ROCSPARSE_CLIENTS_WITH_ROCTX )
+    message(STATUS "ROCTX tracing support enabled for target rocsparse-bench.")
+  else()
+    message(WARNING "ROCTX tracing will be disabled for target rocsparse-bench.")
+
+    if(NOT ROCTX_INCLUDE_DIR)
+      message(WARNING "Header 'roctracer/roctx.h' not found.")
+    endif()
+
+    if(NOT ROCTX_LIBRARY)
+      message(WARNING "Library 'roctx64' not found.")
+    endif()
```

Comment:
You probably don't need to print these warnings. If users need more information about why roctx was not found, they can use [`-DCMAKE_FIND_DEBUG_MODE=ON`](https://cmake.org/cmake/help/latest/variable/CMAKE_FIND_DEBUG_MODE.html) or [`--trace-source=<project>/cmake/Findroctx.cmake`](https://cmake.org/cmake/help/v4.0/manual/cmake.1.html).